src/facade/dragonfly_connection.h

```diff
@@ -19,6 +19,7 @@
 #include "core/fibers.h"
 #include "facade/facade_types.h"
 #include "facade/resp_expr.h"
+#include "util/tls/tls_socket.h"
```

Comment:
why did you need it here?

Reply:
Oh it's `FiberSocketBase *`, I completely missed that, I thought it's used explicitly. removed.